dart/gui/GlutWindow.hpp

```diff
@@ -33,67 +33,17 @@
 #ifndef DART_GUI_GLUTWINDOW_HPP_
 #define DART_GUI_GLUTWINDOW_HPP_
 
-#include <vector>
+#include "dart/gui/glut/GlutWindow.hpp"
 
-#include "dart/gui/LoadOpengl.hpp"
-#include "dart/gui/RenderInterface.hpp"
+#warning "This file is deprecated in DART 6.6. "\
```

Comment:
It seems this method for warning [is not portable](https://stackoverflow.com/a/279669).

We might want to add a `DART_PREPROCESSOR_WARNING(X)` macro to `dart/common/Deprecated.hpp` in order to have a cross-platform method for this.

Reply:
After some more thought, a preprocessor directive can't replaced with a macro because of the limitations of the preprocessor.

However, it seems recent versions of GCC and Clang can understand `#pragma message`, so we can use that instead.